### PR TITLE
Update custom-branding.md

### PR DESCRIPTION
**Pull request by EDIflyer**
Clarify required folder structure for custom icons and reminder re those using Docker - based on what I found on [#2039](https://github.com/filebrowser/filebrowser/issues/2039)

### CODE REVIEW

File: configuration/custom-branding.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 filebrowser config set --branding.name "My Name" \
     --branding.files "/abs/path/to/my/dir" \
     --branding.disableExternal
 ```
+Or can be set under 'Branding directory path' in **Settings → Global Settings**. 
+{% hint style="info" %} If using Docker then remember to bind this directory, for example as `/home/username/containers/filebrowser/branding:/branding` {% endhint %}
 
-Or you can go to **Settings → Global Settings** and change it there.
+For custom icons to be recognised you need to create `img` and `img/icons` directories and place the svg in the `branding/img` directory:
+![](https://user-images.githubusercontent.com/13610277/179242249-3d0554ab-508d-4533-8957-f3c38bb02258.png)
```
Comment (ramiresviana) [inline configuration/custom-branding.md:21]: The images used on this documentation are placed at `.gitbook/assets/`

File: configuration/custom-branding.md [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
 
+To replace the favico you need to place this in the `img/icons` directory but also note that some of the other PNG icon types will be required too (see the default logotypes link above) as the browser will normally use the highest resolution option available (at a minimum the 16x16 and 32x32 options).  You can use the [Real Favicon Generator](https://realfavicongenerator.net/) to generate these for you from your base image.
```
Comment (ramiresviana) [inline configuration/custom-branding.md:23]:
```suggestion
To replace the favicon you need to place this in the `img/icons` directory but also note that some of the other PNG icon types will be required too (see the default logotypes link above) as the browser will normally use the highest resolution option available (at a minimum the 16x16 and 32x32 options).  You can use the [Real Favicon Generator](https://realfavicongenerator.net/) to generate these for you from your base image.
```

**Review by ramiresviana — CHANGES_REQUESTED** [review]

File: configuration/custom-branding.md (hunk 1 above)
Comment (EDIflyer) [reply configuration/custom-branding.md]: OK have added to there and updated code

**Comment by EDIflyer** [comment]
Thanks @ramiresviana - should all be sorted now. Do you know if there is an example custom CSS file somewhere? If so it would be good to link to that too to help users understand what they could change.

**Pull request merged**